tool/tctl/common/resource_command.go

```diff
@@ -742,3 +774,7 @@ func UpsertVerb(exists bool, force bool) string {
 		return "unknown"
 	}
 }
+
+const managedByStaticMsg = `This resource is managed by static configuration. We recommend removing configuration from teleport.yaml, restarting the servers and trying this command again.
+
+If you would still like to proceed, re-run the command with both --force and --confirm flags.`
```

Comment:
nit: inline this error message above

Reply:
It should be okay to leave this as-is since there may be more dynamic config resources coming, all of which could then refer to this message?